internal/models/cleanup.go

```diff
@@ -55,14 +55,21 @@ func (c *Cleanup) Setup() {
 		fmt.Sprintf("delete from %q where id in (select id from %q where created_at < now() - interval '24 hours' and status = 'unverified' limit 100 for update skip locked);", tableMFAFactors, tableMFAFactors),
 	)
 
-	if c.SessionTimebox != nil {
-		timeboxSeconds := int((*c.SessionTimebox).Seconds())
+	if config.External.AnonymousUsers.Enabled {
+		// delete anonymous users older than 30 days
+		c.cleanupStatements = append(c.cleanupStatements,
+			fmt.Sprintf("delete from %q where id in (select id from %q where created_at < now() and is_anonymous is true limit 100 for update skip locked);", tableUsers, tableUsers),
```

Comment:
Do we need to include the interval?

```suggestion
			fmt.Sprintf("delete from %q where id in (select id from %q where created_at < now() - interval '30 days' and is_anonymous is true limit 100 for update skip locked);", tableUsers, tableUsers),
```